lightcurve/views.py

```diff
@@ -1,24 +1,36 @@
 from django.http import JsonResponse
-from django.shortcuts import render_to_response
+from django.shortcuts import render_to_response, get_object_or_404
 from django.template import RequestContext
-from django.shortcuts import get_object_or_404
 
 from astrometry.models import AstrometrySubmission
 from astrometry.process import process_astrometry_online
 from corrections import get_jd_for_analysis
 from imageflow.models import ImageAnalysis, ImageFilter, Reduction, UserUploadedImage
 from lightcurve.models import LightCurve
 from reduction.util import find_point_by_id
+from lightcurve.util import ordered_analysis_status
```

Comment:
Let's move this up one line, so it's below the `lightcurve.models` import

Reply:
Done